imports/ui/explorer-components/item-row.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react'
+import UnitTypeIcon from '../unit-explorer/unit-type-icon'
+import { ItemList } from '../explorer-components/item-list'
+import { Link } from 'react-router-dom'
+import { storeBreadcrumb } from '../general-actions'
+import PropTypes from 'prop-types'
+
+export class ItemRow extends Component {
```

Comment:
The name of "ItemRow" is a bit confusing, as it clearly represents a **unit** and not just any generic item. It probably doesn't matter, as I'll explain next about the need for "UnitGroupList"